src/client/components/forms/publisher.js

```diff
@@ -47,6 +59,11 @@ class PublisherForm extends React.Component {
 		this.handleSubmit = this.handleSubmit.bind(this);
 	}
 
+	/**
+	 * Changes to the tab selected by the user, which has the number
+	 * in 'tab', and checks the validity of the alias and other data.
+	 * @param {number} tab - the number of the selected tab.
```

Comment:
I think you can exclude the latter 2 portions of this description. Avoid mentioning behavior that is not included in this handler.